packages/SwingSet/src/controller/startXSnap.js

```diff
@@ -0,0 +1,96 @@
+import fs from 'fs';
+import path from 'path';
+import { Fail } from '@agoric/assert';
+import { type as osType } from 'os';
+import { xsnap, recordXSnap } from '@agoric/xsnap';
+
+const NETSTRING_MAX_CHUNK_SIZE = 12_000_000;
+
+/**
+ * @param {{ moduleFormat: string, source: string }[]} bundles
+ * @param {{
+ *   snapStore?: SnapStore,
+ *   spawn: typeof import('child_process').spawn
+ *   debug?: boolean,
+ *   traceFile?: string,
```

Comment:
Can we come up with a better name than `traceFile` for this option? Maybe something like `rootPath`?

Reply:
hm, the argument name needs to indicate two things:
* this is a path to a directory, which will be populated with subdirectories (one per `xsnap` launched), each of which will be populated with files (one per command/response exchanged with the worker)
* the files written there are a trace of interactions with the worker

How about `workerTraceRootPath` ?

Reply:
yeah, that reads pretty well, I'll use `workerTraceRootPath` and `makeNextTraceDir()`